src/victory-transition/victory-transition.js

```diff
@@ -87,7 +89,9 @@ export default class VictoryTransition extends React.Component {
     const child = React.Children.toArray(props.children)[0];
     const transitionProps = getTransitionProps(child);
     const domain = {
-      x: this.getDomainFromChildren(props, "x"),
+      x: this.state && this.state.nodesWillExit
+        ? this.getDomainFromChildren(this.props, "x")
```

Comment:
Nope, we will update the only the x axis in exiting, the lines will only translate along the x axis, which will be cover by clipPath, and then exit after it couldn't be seen.